test/github_scope_token_to_list_of_private_public_repos_test.go

```diff
@@ -28,9 +28,9 @@ import (
 )
 
 func TestGithubPullRequestScopeTokenToListOfRepos(t *testing.T) {
-	if os.Getenv("NIGHTLY_E2E_TEST") != "true" {
-		t.Skip("Skipping test since only enabled for nightly")
-	}
+	//if os.Getenv("NIGHTLY_E2E_TEST") != "true" {
+	//	t.Skip("Skipping test since only enabled for nightly")
+	//}
```

Comment:
Commented to make sure e2e runs successfully for this PR to make sure fix is proper
will revert back once e2e is success